From a1b66ebcf35f65a72fec5ea57dcecaa2d1d160a1 Mon Sep 17 00:00:00 2001
From: Leoni Prodduvaka <leoni.prodduvaka@intel.com>
Date: Tue, 18 Apr 2017 22:57:51 +0530
Subject: [PATCH 0605/1443] ASoC: Intel: CNL: Add support for ICL FPGA SDW
 Aggregation

Added Master 1 as the skl_platform_dai id for icl fpga sdw aggregation

Change-Id: Ia21d008dddad0ca0928b7903d6760e0da0433b04
Signed-off-by: Leoni Prodduvaka
Reviewed-on:
Reviewed-by: audio_build
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Reviewed-by: R, Dharageswari <dharageswari.r@intel.com>
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Reviewed-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 sound/soc/intel/skylake/skl-pcm.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/sound/soc/intel/skylake/skl-pcm.c b/sound/soc/intel/skylake/skl-pcm.c
index 2da571f..6ad9937 100644
--- a/sound/soc/intel/skylake/skl-pcm.c
+++ b/sound/soc/intel/skylake/skl-pcm.c
@@ -1330,10 +1330,14 @@ static struct snd_soc_dai_driver skl_platform_dai[] = {
 	 */
 	.name = "SDW Pin",
 #if IS_ENABLED(CONFIG_SND_SOC_INTEL_CNL_FPGA)
+#ifndef CONFIG_SND_SOC_SDW_AGGM1M2
 	.id = SDW_BE_DAI_ID_MSTR0,
 #else
 	.id = SDW_BE_DAI_ID_MSTR1,
 #endif
+#else
+	.id = SDW_BE_DAI_ID_MSTR1,
+#endif
 	.ops = &skl_sdw_dai_ops,
 	.playback = {
 		.stream_name = "SDW Tx",
@@ -1356,10 +1360,14 @@ static struct snd_soc_dai_driver skl_platform_dai[] = {
 	 */
 	.name = "SDW10 Pin",
 #if IS_ENABLED(CONFIG_SND_SOC_INTEL_CNL_FPGA)
+#ifndef CONFIG_SND_SOC_SDW_AGGM1M2
 	.id = SDW_BE_DAI_ID_MSTR0,
 #else
 	.id = SDW_BE_DAI_ID_MSTR1,
 #endif
+#else
+	.id = SDW_BE_DAI_ID_MSTR1,
+#endif
 	.ops = &skl_sdw_dai_ops,
 	.playback = {
 		.stream_name = "SDW Tx10",
-- 
2.7.4

